src/css/gameboard.styl

```diff
@@ -101,15 +101,18 @@
 
     .counter
         padding: 0
-        font-size: 12px
-        width: 24px
-        height: 24px
-        line-height: 24px
+        font-size: 1rem
+        width: 2rem
+        height: 2rem
+        border-radius: 100%
```

Comment:
is this supposed to be included?

Reply:
Nope! That was a leak from an experiment I was doing with tokens. 😆